packages/block-library/src/navigation/style.scss

```diff
@@ -216,6 +218,16 @@ $navigation-icon-size: 24px;
 				}
 			}
 
+			// Push inwards from right edge of submenu.
+			.wp-block-navigation__submenu-icon {
+				margin-right: 0.25em;
+			}
+
+			// For hover-based menus, this class is duplicated for a button.
+			.wp-block-navigation__submenu-icon .wp-block-navigation__submenu-icon {
```

Comment:
Is this still needed? I notice you removed the extra span [here](https://github.com/WordPress/gutenberg/pull/37003/files#diff-2de79e33fab32eada85be3d008e9d8a124940b1b42e8bccb345925051e57bd74R200) (which wasn't meant to be there, so thanks for fixing 😄 )

Reply:
Wildly excellent eye there. I started writing the CSS rule first, then figured the better fix was to remove the span, then forgot to remove the CSS 🙈 

Thank you, removed!